chapters/ch51.md

```diff
@@ -1,101 +1,102 @@
 > # 51. Choosing pipeline components: Task simplicity
 
---> _replace THIS LINE by your translation for the above line_
+# 51. Lựa chọn các thành phần cho pipeline: tính đơn giản của tác vụ
 
 > Other than data availability, you should also consider a second factor when picking components of a pipeline: How simple are the tasks solved by the individual components? You should try to choose pipeline components that are individually easy to build or learn. But what does it mean for a component to be "easy" to learn?
 
---> _replace THIS LINE by your translation for the above line_
+Ngoài sự sẵn có của dữ liệu, bạn cũng nên xem xét một nhân tố thứ hai khi lựa chọn các thành phần của một pipeline: Việc giải quyết từng tác vụ bằng các thành phần riêng lẻ đơn giản đến mức nào? Bạn nên cố gắng chọn những thành phần pipeline có thể dễ dàng xây dựng hay học riêng lẻ. Nhưng các thành phần "dễ" học nghĩa là gì?
 
 ![img](../imgs/C51_01.png)
 
 > Consider these machine learning tasks, listed in order of increasing difficulty:
 
---> _replace THIS LINE by your translation for the above line_
+Xét những tác vụ học máy này, được liệt kê theo thứ tự độ khó tăng dần:
 
 > 1. Classifying whether an image is overexposed (like the example above)
 
---> _replace THIS LINE by your translation for the above line_
+1. Phân loại một bức ảnh có bị phơi sáng quá mức hay không (như trong ví dụ trên)
 
 > 2. Classifying whether an image was taken indoor or outdoor
 
---> _replace THIS LINE by your translation for the above line_
+2. Phân loại một bức ảnh được chụp trong nhà hay ngoài trời
 
 > 3. Classifying whether an image contains a cat
 
---> _replace THIS LINE by your translation for the above line_
+Phân loại một bức ảnh có chứa một con mèo hay không
 
 > 4. Classifying whether an image contains a cat with both black and white fur
 
---> _replace THIS LINE by your translation for the above line_
+Phân loại liệu một bức ảnh chứa một con mèo khoang đen trắng hay không
 
 > 5. Classifying whether an image contains a Siamese cat (a particular breed of cat)
 
---> _replace THIS LINE by your translation for the above line_
+Phân loại liệu một bức ảnh có chứa một con mèo Siamese (tên một loại mèo)
 
 > Each of these is a binary image classification task: You have to input an image, and output either 0 or 1. But the tasks earlier in the list seem much "easier" for a neural network to learn. You will be able to learn the easier tasks with fewer training examples.
 
---> _replace THIS LINE by your translation for the above line_
+Mỗi tác vụ trong số này là một bài toán phân loại ảnh nhị phân: Bạn phải có một bức ảnh đầu vào, và cho ra hoặc 0 hoặc 1. Nhưng những tác vụ đầu tiên trong danh sách này có vẻ "dễ" hơn nhiều để một mạng neural có thể học. Bạn sẽ có thể huấn luyện những tác vụ này với ít mẫu huấn luyện hơn.
```

Comment:
- Mỗi tác vụ ~~trong số này~~ **_trên_** là một bài toán phân loại ảnh nhị phân

Reply:
- bỏ một từ : ~~hoặc~~ 0 hoặc 1

Reply:
- Nhưng những tác vụ đầu tiên trong danh sách này có vẻ **_quá_** "dễ" ~~hơn nhiều để một~~ **_đối với/cho_** mạng neural ~~có thể học~~
- Câu cuối tác giả ý nói chung hơn về tác vụ dễ chứ không phải là "tác vụ này". ---> 

> "Bạn sẽ có thể học tác vụ dễ hơn với ít lượng mẫu huấn luyện hơn"